Assets/Scripts/PlayerDataManager.cs

```diff
@@ -0,0 +1,55 @@
+using UnityEngine;
+
+namespace VectorSchool.DataLayer
```

Comment:
Please make sure your namespce  structure match the folders structure. This script should be under "DataLayer" folder then

Reply:
Very good implementation. Great job